src/interpreter.h

```diff
@@ -10,13 +10,24 @@
 
 #ifndef _interpreter_h_
 #define _interpreter_h_
+
+#include "perfmon.h"
 /*
 #define ACMD(name)  \
    void (name)(struct char_data *ch, __attribute__ ((unused)) char *argument, __attribute__ ((unused)) int cmd, __attribute__ ((unused)) int subcmd)
 */
 
-#define ACMD(name)   \
+#define ACMD_DECLARE(name)   \
```

Comment:
What is the purpose of changing ACMD() to ACMD_DECLARE() and adding instrumentation to a new version of ACMD()? Should all new commands be written with ACMD_DECLARE() now?

Reply:
I understand the change now-- ACMD_DECLARE() for prototypes, ACMD() for actual commands.